src/main/groovy/life/qbic/controller/SamplesController.groovy

```diff
@@ -70,8 +71,13 @@ class SamplesController {
   HttpResponse<Location> newLocation(@PathVariable('sampleId') String sampleId, Location location) {
     if(!RegExValidator.isValidSampleCode(sampleId)) {
       return HttpResponse.status(HttpStatus.BAD_REQUEST, "Not a valid sample code!")
-    } else {
-      return sampleService.addNewLocation(sampleId, location)
+    }
+    try{
+      sampleService.addNewLocation(sampleId, location)
+      return HttpResponse.created(location)
```

Comment:
For making the test work this response needs to be CREATED but was stated as OK in the DB manager (see other comment). Is it OK to change it here or do I need to change the test?